test/e2e/kubevirt/types.go

```diff
@@ -1,7 +1,8 @@
 package kubevirt
 
 const (
-	FedoraCoreOSContainerDiskImage = "quay.io/kubevirtci/fedora-coreos-kubevirt:v20230905-be4fa50"
-	FedoraContainerDiskImage       = "quay.io/kubevirtci/fedora-with-test-tooling:v20241128-4d4c8fe"
-	FakeLauncherImage              = "quay.io/nmstate/c10s-nmstate-dev:latest"
+	FedoraCoreOSContainerDiskImage          = "quay.io/kubevirtci/fedora-coreos-kubevirt:v20230905-be4fa50"
+	FedoraWithTestToolingContainerDiskImage = "quay.io/kubevirtci/fedora-with-test-tooling:v20241128-4d4c8fe"
+	FedoraContainerDiskImage                = "quay.io/containerdisks/fedora:39"
```

Comment:
Please add an explanation for moving to F39 in the commit message.
Is there any bug tracking the RA with lifetime 0 issue in Fedora? That would be useful as well.

Reply:
I mean ... we were running fc35 - which is not supported since 2022-12-13 per the [fedora docs](https://docs.fedoraproject.org/en-US/releases/eol/).

Actually, even fc39 is currently unsupported. Couldn't we use 40 instead ?

Reply:
> I mean ... we were running fc35 - which is not supported since 2022-12-13 per the [fedora docs](https://docs.fedoraproject.org/en-US/releases/eol/).
> 
> Actually, even fc39 is currently unsupported. Couldn't we use 40 instead ?

If we use fedora40 then we need either https://github.com/kubevirt/kubevirt/pull/13458 (not present at v1.4.0 kubevirt) or disable the NetworkManager acd mechanism

Reply:
> I mean ... we were running fc35 - which is not supported since 2022-12-13 per the [fedora docs](https://docs.fedoraproject.org/en-US/releases/eol/).

Sure that's a valid reason, would be good to have it in the commit message.
I don't think that was the case here though? It would be great to have a tracker for the issue @qinqon mentioned.

Reply:
> > I mean ... we were running fc35 - which is not supported since 2022-12-13 per the [fedora docs](https://docs.fedoraproject.org/en-US/releases/eol/).
> 
> Sure that's a valid reason, would be good to have it in the commit message. I don't think that was the case here though? It would be great to have a tracker for the issue @qinqon mentioned.

Also I was thinking that would be good to backport https://github.com/kubevirt/kubevirt/pull/13458 to openshift virt 4.18 since this is more or less GA for primary UDNs ?

Reply:
Ahh we have kubevirt/kubevirt 1.5.0 hours ago, that should contain the ignore arp fix, let me try fedora 40, but still we need to backport the fix to 1.4 so it lands on openshift virt 4.18

Reply:
> > I mean ... we were running fc35 - which is not supported since 2022-12-13 per the [fedora docs](https://docs.fedoraproject.org/en-US/releases/eol/).
> 
> Sure that's a valid reason, would be good to have it in the commit message. I don't think that was the case here though? It would be great to have a tracker for the issue @qinqon mentioned.

@kyrtapz I don't find the bug since it's quite old stuff, is this mandatory ?